app/helpers/plain-date.js

```diff
@@ -0,0 +1,12 @@
+import { helper } from '@ember/component/helper';
+import lightFormat from 'date-fns/lightFormat';
+
+export default helper(function plainDate([datetime]/*, hash*/) {
+  try {
+    return lightFormat(datetime, 'dd/MM/yyyy');
```

Comment:
In OP we created a helper that uses [`Intl.Datetimeformat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) (with hardcoded country since we didn't have other requirements).
Maybe something similar can be done here? Or do you prefer manual formatting?

Reply:
on testing I ended up with a different separator (I think `-`) and `/` was preferred here. I can recheck with customer if that's still important

Reply:
Not entirely: formatting as 'dd/MM/yyyy' would give '21/01/2022' and Intl gives '21/1/2022', unless you use `Intl.DateTimeFormat("nl-BE", { dateStyle: "short" }).format(datetime);`. Weird, but consistent across browsers.